Extensions/TopDownMovementBehavior/TopDownMovementBehavior.cpp

```diff
@@ -46,47 +46,55 @@ TopDownMovementBehavior::GetProperties(
     const gd::SerializerElement& behaviorContent) const {
   std::map<gd::String, gd::PropertyDescriptor> properties;
 
-  properties[_("Allows diagonals")]
+  properties["AllowDiagonals"]
```

Comment:
Any thoughts on using camelCase rather than PascalCase? I think some behaviors already use camelCase for properties, because they are like a, well, property on an object (`myObject.myProperty` in code)

Reply:
This is the good practice for community extensions. It's better if every extensions use the same format. It avoids to care about the case in code handling property name most of the time.

Reply:
Alright then. At some point casing should not be an issue anyway, it's the kind of stuff we wonder about as developers but that should be a non issue entirely.